tests/test_ext_marshmallow.py

```diff
@@ -51,7 +51,7 @@ def resolver(schema):
             openapi_version="2.0",
             plugins=(MarshmallowPlugin(schema_name_resolver=resolver),),
         )
-        assert {} == get_schemas(spec)
+        assert get_schemas(spec) is None
```

Comment:
Nit: It might be good for the `get_` functions to default to a sentinel value, like `marshmallow.missing`.

Reply:
I'd rather avoid `missing` as this could pass a wrong change leaking the `missing` object into the generated spec.

I could create a dedicated `MISSING_FROM_SPEC` singleton.

Overall, it looks just as simple to catch `KeyError`. Also, when using OAS3, `KeyError` covers the case where the whole `components` section is missing. See new commit.

Reply:
Works for me.